llama-index-integrations/vector_stores/llama-index-vector-stores-mongodb/pyproject.toml

```diff
@@ -31,7 +31,7 @@ version = "0.1.4"
 
 [tool.poetry.dependencies]
 python = ">=3.8.1,<4.0"
-llama-index-core = "^0.10.1"
+llama-index = "^0.10.1"
```

Comment:
Could minimize this change. Could leave with core, and then add the bare minimum in dev dependencies. I think I just needed embeddings-openai